src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -357,55 +346,137 @@ def gui_submit(bundle_directory: str) -> Optional[dict[str, Any]]:
         return None
 
 
+def _options_dialog() -> dict[str, Any]:
+    """
+    Builds and displays a dialog within KeyShot to get the submission options
+    reuired before the main gui submission window is opened outside of KeyShot.
+    Options:
+        Option 1: Dropdown to select whether to submit just the scene file itself
+                  or all external file references as well by packing/unpacking a
+                  KSP bundle before submission.
+    Returns a dictionary of the selected option values in the format:
+        {'SUBMISSION_MODE_KEY': [1, 'only the scene BIP file']}
+    """
+    dialog_items = [
+        (
+            SUBMISSION_MODE_KEY,
+            lux.DIALOG_ITEM,
+            "Do you want to submit",
+            0,
+            ["the scene BIP file and all external files references", "only the scene BIP file"],
```

Comment:
I think the wording could be tweaked a bit so an artist would know how to choose one setting versus another. Maybe include a description like:
> Choose "the scene BIP file and all external file references" if you're rendering on service-managed fleets or to upload your scene to a customer-managed fleet. Choose "only the scene BIP file" if you're rendering on customer-managed fleets that already have access to the file references, for example on a shared network drive. 

Nit: Also instead of "Do you want to submit", maybe rephrase as "What files do you want to attach with your job?" That way the user selection answers the question instead of form part of the question itself.

Reply:
There's nothing stopping someone from setting up network storage for service managed fleets as far as I know. I don't think we can make the suggestion of which to use based on if SMF or CMF are being used.

Reply:
Updated the wording of the prompt itself